### PR TITLE
Extend data

### PR DESCRIPTION
**Pull request by ycaokris**

### CODE REVIEW

**Comment by ycaokris** [comment]
@shammamah  @Marc-Andre-Rivet  A big shout out to `extendData` property, this solves my network traffic problem caused by simultaneous updating multiple time-series plots.

Screen Shot 2019-04-11 at 2 52 29 PM

Screen Shot 2019-04-11 at 11 05 50 PM

File: app.py [hunk 1]
```diff
@@ -655,7 +637,7 @@ def build_chart_panel():
 
             dcc.Interval(
                 id='interval-component',
-                interval=2 * 1000,  # in milliseconds
+                interval=1 * 1000,  # in milliseconds
```
Comment (shammamah) [inline app.py]: Nice that we can speed things up now! :)

File: requirements.txt [hunk 2]
```diff
@@ -2,11 +2,11 @@ attrs==19.1.0
 certifi==2019.3.9
```
Comment (shammamah) [inline requirements.txt]: Change these to `>=`

Comment (ycaokris) [reply requirements.txt]: Done.

**Pull request merged**